# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-11-19 04:01-0600\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: exceptions.py:23
msgid "Something went wrong"
msgstr "Midagi läks valesti"

#: exceptions.py:51
#, fuzzy
#| msgid "A signing session is already in progress."
msgid "Operation is in progress."
msgstr "Allkirjastamise sessioon on juba käsil"

#: exceptions.py:69
#, python-brace-format
msgid "The {service} service reported a failure."
msgstr "Teenus {service} teatas ebaõnnestumisest."

#: exceptions.py:77
#, python-brace-format
msgid ""
"The {service} service is unavailable at the moment. Please try again later."
msgstr ""
"Teenus {service} ei ole hetkel kättesaadav. Palun proovige hiljem uuesti."

#: exceptions.py:85
msgid "Operation with user certificate level ADVANCED not allowed."
msgstr "Operatsioon kasutajasertifikaadi tasemega ADVANCED ei ole lubatud"

#: exceptions.py:127
msgid "A signing session is already in progress."
msgstr "Allkirjastamise sessioon on juba käsil"

#: exceptions.py:133
msgid "No signing session found."
msgstr "Allkirjastamise sessiooni ei leitud"

#: exceptions.py:144
msgid "Invalid parameters."
msgstr "Vigased parameetrid."

#: exceptions.py:153
#, python-brace-format
msgid "Invalid value for parameter {param}."
msgstr "Vigane väärtus parameetri {param} jaoks."

#: exceptions.py:163
msgid "Invalid ID code."
msgstr "Vale isikukood."

#: exceptions.py:174
msgid "User not registered in the service."
msgstr "Kasutaja pole registreeritud."

#: exceptions.py:182
msgid "The signing operation was canceled by the user."
msgstr "Allkirjastamine tühistati kasutaja poolt."

#: exceptions.py:195
msgid "The signing operation timed out."
msgstr "Allkirjastamise operatsiooni ajaaken aegus."

#: exceptions.py:206
msgid "The container has been already signed by the user."
msgstr "Antud Konteiner on juba selle kasutaja poolt allkirjastatud."

#: mobileid/i18n.py:13 smartid/i18n.py:12
msgid "Authenticate"
msgstr "Isikut tuvastama"

#: mobileid/i18n.py:14 smartid/i18n.py:13
msgid "Sign"
msgstr "Allkirjasta"

#: signing/views.py:190
msgid "Internal server error"
msgstr "Viga serveri sisemuses"

#: smartid/apps.py:7
msgid "SmartIDService"
msgstr "Smart-ID Teenus"
